egs/chime4/s5_1ch/local/compute_PESQ.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Copyright 2017 Johns Hopkins University (Author: Aswin Shanmugam Subramanian)
```

Comment:
I notice a lot of scripts in this PR use a mixture of lower and upper case for their names.
Could you please change all of them to all-lowercase, unless there is a very compelling reason?
We always use lowercase for names of scripts.
The same goes for variable names in bash scripts: I see a lot of mixed-case there too.  E.g. change tMOS to t_mos